org.openomf.OpenOMF.json

```diff
@@ -0,0 +1,191 @@
+{
+  "app-id": "org.openomf.OpenOMF",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openomf",
+  "permissions": [
```

Comment:
where in the documentation does this say this is valid? So I can get it fixed.

Reply:
I don't know, should I just remove it? Is access to `XDG_STATE_HOME` and `XDG_CONFIG_HOME` always available?

Reply:
> I don't know, should I just remove it?

How can you say you don't know? Either you saw it in some documentation that is wrong, or you just made it up, and in that case it has no business being here. 

*hint* It's invalid, and you probably ignored it when flatpak-builder told you so.

Reply:
Removed.